index-en.html

```diff
@@ -0,0 +1,849 @@
+                                        <div class="row">
+                                          <div class="col-sm-12 text-center fact-inner scrollto">
+                                            <a href="#newsletter">
+                                              <button style="float: none;" type="button" class="btn btn-theme-color btn-lg">Subscribe to the newsletter</button>
+                                            </a>
+                                            </span>
+                                          </div>
+                                        </div>
+                                    </p>
+                                </div>
+                            </div><!--about us box end-->
+                        </div>
+                        <div class="col-md-6 col-sm-6">
+                            <div class="about-box wow animated fadeInDown" data-wow-delay="0.5">
+                                <div class="about-box-icon">
+                                    <i class="fa fa-code"></i>
+                                </div>
+                                <div class="about-box-text">
+                                    <h4>Our mission</h4>
+                                    <p align="justify">
+                                        Our core mission is the create a strong community of students who share a passion for computer science. Such a community allows us to help our members reach their goals, whether they're about skillset development, professional development, involvement, personal projects, general discoveries, and more!
+                                    </p>
+                                </div>
+                            </div><!--about us box end-->
+                        </div>
+                    </div><!--about services row end-->
+                    <div class="row">
+                        <div class="col-md-6 col-sm-6">
+                            <div class="about-box wow animated fadeInDown" data-wow-delay="0.5s">
+                                <div class="about-box-icon">
+                                    <i class="fa fa-image"></i>
+                                </div>
+                                <div class="about-box-text">
+                                    <h4>Our vision</h4>
+                                    <p align="justify">
+                                        We want to be at the center of everything related to computer science, here at the main campus of Université de Sherbrooke. We are a catalyst for events, participation and implication, as well as an accelerator to help our members reach the industry's standards of excellency.
```

Comment:
I was wondering whether we should keep the original name (Université de Sherbrooke) or translate it loosely (University of Sherbrooke / Sherbrooke University). That would help with the rest, such as "campus of [the] Sherbrooke University", for example.